src/components/ToDoItemEdit.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div
+    class="relative sm:mx-auto mx-[16px] flex flex-col space-y-8 justify-center rounded-2xl bg-white border-black border-2 p-[18px] m-[50px] sm:w-[610px] w-[320px]"
+  >
+    <div class="flex w-full justify-between flex-row">
+      <div class="flex flex-col">
+        <input
+          class="w-full placeholder-black leading-[3rem] sm:text-5xl text-lg flex mt-4 font-bold"
+          type="text"
+          placeholder="Title"
+        />
+
+        <div class="flex w-full items-center sm:h-[12px] h-[5px] lg:hidden">
+          <img :src="dateIcon" class="lg:hidden mr-1 h-3 w-3" />
+
+          {{ localTodo.createdAt }}
+        </div>
+      </div>
+
+      <ToDoPriority v-model:priority="localTodo.priority" />
+      <div class="flex flexf-col sm:invisible visible">
+        <BaseButton
+          class="flex justify-center flex-nowrap bg-green-500 rounded-2xl w-2.5 h-2 ml-2"
+          @click="setPriority(priorities.High)"
+        />
+
+        <BaseButton
+          class="flex justify-center flex-nowrap bg-yellow-500 rounded-2xl w-2.5 h-2 ml-2"
+          @click="setPriority(priorities.Medium)"
+        />
+        <BaseButton
+          class="flex justify-center flex-nowrap bg-red-500 rounded-2xl w-2.5 h-2 ml-2"
+          :is-highlihted="false"
```

Comment:
Why do you use this `is-highlihted` prop only here? If is not relevant please remover or make sure you highlight in the right way (based on the design) the active/selected priority